libs/libcommon/src/libcommon/constants.py

```diff
@@ -14,7 +14,7 @@
 PROCESSING_STEP_DATASET_SIZE_VERSION = 1
 PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 1
 PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_STREAMING_VERSION = 1
-PROCESSING_STEP_FIRST_ROWS_VERSION = 2
+PROCESSING_STEP_FIRST_ROWS_FROM_STREAMING_VERSION = 2
```

Comment:
I didn't change the version here since it is just a `kind` renaming change and a fix for split validation.
If I upgrade this version, `cache-refresh` job could start appending new jobs for the outdated records, but that is not the intention here at least we want to re compute all the first rows again.